src/OSSGadget.sln

```diff
@@ -85,6 +87,10 @@ Global
 		{23D17E08-9E95-4448-B75D-BABAF1F3C854}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{23D17E08-9E95-4448-B75D-BABAF1F3C854}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{23D17E08-9E95-4448-B75D-BABAF1F3C854}.Release|Any CPU.Build.0 = Release|Any CPU
+		{17C9F321-8A49-4140-B009-AD476F959D42}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Exception here...

E:\root\OSSGadget\src\oss-diff\bin\Debug\net5.0>.\oss-diff.exe -f sarifv2 -O temp/sarif.sarif pkg:cargo/rand@0.8.3  pkg:cargo/rand@0.8.2
Unhandled exception. System.IO.DirectoryNotFoundException: Could not find a part of the path 'E:\root\OSSGadget\src\oss-diff\bin\Debug\net5.0\temp\sarif.sarif'.
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.StreamWriter.ValidateArgsAndOpenPath(String path, Boolean append, Encoding encoding, Int32 bufferSize)
   at System.IO.StreamWriter..ctor(String path)
   at System.IO.File.WriteAllText(String path, String contents)
   at Microsoft.CST.OpenSource.DiffTool.DiffTool.RunAsync(Options options) in E:\root\OSSGadget\src\oss-diff\DiffTool.cs:line 428
   at CommandLine.ParserResultExtensions.WithParsedAsync[T](ParserResult`1 result, Func`2 action)
   at Microsoft.CST.OpenSource.DiffTool.DiffTool.Main(String[] args) in E:\root\OSSGadget\src\oss-diff\DiffTool.cs:line 84
   at Microsoft.CST.OpenSource.DiffTool.DiffTool.<Main>(String[] args)

Reply:
btw, temp doesnt exist. error is probably because of assuming full path exists

Reply:
Fixed.